conf/application.conf

```diff
@@ -119,46 +114,15 @@ controllers {
 # ~~~~~
 # You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
 
-# Metrics plugin settings - graphite reporting is configured on a per env basis
-metrics {
-    name = ${appName}
-    rateUnit = SECONDS
-    durationUnit = SECONDS
-    showSamples = true
-    jvm = true
-    enabled = true
-}
-
-
 # Microservice specific config
 
-auditing {
-    enabled=true
-    traceRequests=true
-    consumer {
-        baseUri {
-            host = localhost
-            port = 8100
-        }
-    }
-}
-
-
 mongodb {
     ninoMatchTtlInSeconds = 18000
     uri = "mongodb://localhost:27017/individuals-matching-api"
 }
 
 
 microservice {
-    metrics {
-        graphite {
-            host = graphite
-            port = 2003
-            prefix = play.${appName}.
```

Comment:
this needs to remain

Reply:
> this needs to remain

So here you mean that the graphite object must remain - and to just delete all the keys? because it's literally an empty object at that point

Reply:
all but the enabled flag

Reply:
sorry, turns out this block can be removed after all